pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/KafkaPartitionLevelConsumer.java

```diff
@@ -45,6 +46,11 @@ public KafkaPartitionLevelConsumer(String clientId, StreamConfig streamConfig, i
     super(clientId, streamConfig, partition, kafkaSimpleConsumerFactory);
   }
 
+  @VisibleForTesting
```

Comment:
I think that this function should locate in the base class that defines `simpleConsumer`. Can we move this to `KafkaConnectionHandler`?

Reply:
updated in new diff